docs/developer/testing/TESTING.md

```diff
@@ -74,7 +86,15 @@ oc delete namespace $OADP_TEST_NAMESPACE
 
 TODO
 
-## Debugging e2e tests with Visual Studio
+## Debugging
+
+```bash
+make test-e2e-setup
+cat /tmp/test-settings/oadpcreds
```

Comment:
This `/tmp/test-settings/oadpcreds` are not specified elsewhere in the doc so it's hard to tell what it's meant for. ie. is this aws creds? or dpa yaml.

Reply:
I may be wrong, but I think it is the `spec` part of the DPA that will be used for the tests

For IBM, was useful for see what I was doing wrong (bucket-file format, provider being set to ibmcloud, etc)

Reply:
Added some info about how this can help

Reply:
The spec part is actually redefined in the dpa helpers file but based off info in this file. This is because we wanted different plugins/featureflags for CSI and data mover for example.